--- nvidia-1.0a-3.2.0/kwin/Makefile.am.orig	2004-07-05 17:03:34.000000000 +0200
+++ nvidia-1.0a-3.2.0/kwin/Makefile.am	2004-07-24 13:59:04.859558144 +0200
@@ -26,69 +26,69 @@
 
 EXTRA_DIST                = $(lnk_DATA)
 
-tiles.h: pics/avatar.png\
-pics/border-left.png\
-pics/border-right.png\
-pics/bottom-center.png\
-pics/bottom-left.png\
-pics/bottom-right.png\
-pics/caption-large-center.png\
-pics/caption-large-left.png\
-pics/caption-large-right.png\
-pics/caption-small-center-act.png\
-pics/caption-small-center-dis.png\
-pics/caption-small-center.png\
-pics/caption-small-left-act.png\
-pics/caption-small-left-dis.png\
-pics/caption-small-left.png\
-pics/caption-small-right-act.png\
-pics/caption-small-right-dis.png\
-pics/caption-small-right.png\
-pics/close-act.png\
-pics/close-dis.png\
-pics/grabbar-center.png\
-pics/grabbar-left.png\
-pics/grabbar-right.png\
-pics/help-act.png\
-pics/help-dis.png\
-pics/maximize-act.png\
-pics/maximize-dis.png\
-pics/minimize-act.png\
-pics/minimize-dis.png\
-pics/restore-act.png\
-pics/restore-dis.png\
-pics/titlebar-center-act.png\
-pics/titlebar-center-dis.png\
-pics/titlebar-center.png\
-pics/titlebar-left-act.png\
-pics/titlebar-left-dis.png\
-pics/titlebar-left.png\
-pics/titlebar-right1-act.png\
-pics/titlebar-right1-dis.png\
-pics/titlebar-right2-act.png\
-pics/titlebar-right2-dis.png\
-pics/titlebar-right3-act.png\
-pics/titlebar-right3-dis.png\
-pics/titlebar-right-act.png\
-pics/titlebar-right-dis.png\
-pics/titlebar-right.png\
-pics/titlebutton-round-act.png\
-pics/titlebutton-round-dis.png\
-pics/titlebutton-round-huge.png\
-pics/titlebutton-round-large.png\
-pics/titlebutton-round.png\
-pics/titlebutton-square-act.png\
-pics/titlebutton-square-dis.png\
-pics/titlebutton-square-huge.png\
-pics/titlebutton-square-large.png\
-pics/titlebutton-square.png\
-pics/alldesktop-act.png\
-pics/alldesktop-dis.png\
-pics/menu-act.png\
-pics/menu-dis.png\
-pics/noalldesktop-act.png\
-pics/noalldesktop-dis.png\
-pics/titlebutton-transparent.png
+tiles.h: pics/avatar.png \
+    pics/border-left.png \
+    pics/border-right.png \
+    pics/bottom-center.png \
+    pics/bottom-left.png \
+    pics/bottom-right.png \
+    pics/caption-large-center.png \
+    pics/caption-large-left.png \
+    pics/caption-large-right.png \
+    pics/caption-small-center-act.png \
+    pics/caption-small-center-dis.png \
+    pics/caption-small-center.png \
+    pics/caption-small-left-act.png \
+    pics/caption-small-left-dis.png \
+    pics/caption-small-left.png \
+    pics/caption-small-right-act.png \
+    pics/caption-small-right-dis.png \
+    pics/caption-small-right.png \
+    pics/close-act.png \
+    pics/close-dis.png \
+    pics/grabbar-center.png \
+    pics/grabbar-left.png \
+    pics/grabbar-right.png \
+    pics/help-act.png \
+    pics/help-dis.png \
+    pics/maximize-act.png \
+    pics/maximize-dis.png \
+    pics/minimize-act.png \
+    pics/minimize-dis.png \
+    pics/restore-act.png \
+    pics/restore-dis.png \
+    pics/titlebar-center-act.png \
+    pics/titlebar-center-dis.png \
+    pics/titlebar-center.png \
+    pics/titlebar-left-act.png \
+    pics/titlebar-left-dis.png \
+    pics/titlebar-left.png \
+    pics/titlebar-right1-act.png \
+    pics/titlebar-right1-dis.png \
+    pics/titlebar-right2-act.png \
+    pics/titlebar-right2-dis.png \
+    pics/titlebar-right3-act.png \
+    pics/titlebar-right3-dis.png \
+    pics/titlebar-right-act.png \
+    pics/titlebar-right-dis.png \
+    pics/titlebar-right.png \
+    pics/titlebutton-round-act.png \
+    pics/titlebutton-round-dis.png \
+    pics/titlebutton-round-huge.png \
+    pics/titlebutton-round-large.png \
+    pics/titlebutton-round.png \
+    pics/titlebutton-square-act.png \
+    pics/titlebutton-square-dis.png \
+    pics/titlebutton-square-huge.png \
+    pics/titlebutton-square-large.png \
+    pics/titlebutton-square.png \
+    pics/alldesktop-act.png \
+    pics/alldesktop-dis.png \
+    pics/menu-act.png \
+    pics/menu-dis.png \
+    pics/noalldesktop-act.png \
+    pics/noalldesktop-dis.png \
+    pics/titlebutton-transparent.png
 
 
 
